okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -256,11 +254,21 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.upgrade.post\" ], "
         + "    \"type\" : \"internal\" "
         + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/install\","
+        + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.install.get\" ], "
+        + "    \"type\" : \"internal\" "
+        + "   }, {"
         + "    \"methods\" :  [ \"POST\" ],"
         + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/install\","
         + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.install.post\" ], "
         + "    \"type\" : \"internal\" "
         + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/install/{installId}\","
+        + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.install.get\" ], "
```

Comment:
Did you intentionally re-use this permission?

Reply:
yes.